ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/CppPipelineInterfacer.java

```diff
@@ -1,17 +1,35 @@
 package edu.wpi.grip.ui.codegeneration.tools;
 
 import java.io.File;
+import java.io.IOException;
 import java.lang.UnsupportedOperationException;
+import java.net.URISyntaxException;
 
 import org.opencv.core.Mat;
 
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertEquals;
+
 public class CppPipelineInterfacer implements PipelineInterfacer {
   
   static{
     System.loadLibrary("genJNI");
   }
   
   public CppPipelineInterfacer(String libName){
+    try {
+      File cpLoc = new File(CppPipelineInterfacer.class.getResource("tools/realpipe/CMakeLists.txt").toURI()).getParentFile();
```

Comment:
Never mind. Had an extra tools/ since CppPipelineInterfacer is already in the tools package.